client/src/components/Feed/StyledPostPage.js

```diff
@@ -0,0 +1,21 @@
+import { mq } from "constants/theme";
+import styled from "styled-components";
+import PostCard from "./PostCard";
+
+export const StyledPostPage = styled.div`
+  width: 50%;
+  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+    width: 100%;
+    position: relative;
+    overflow-wrap: break-word;
+  }      
+`;
+
+export const StyledPostPagePostCard = styled(PostCard)`
+  max-width: 80rem;
```

Comment:
The only issue I see with this one is if it is a 4k Resolution then it will be tooo small.